### PR TITLE
Feature/car model capacity

### PR DESCRIPTION
**Pull request by Scttpr**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Scttpr** [comment]
[Ajouter un nombre de place pour le modèle de véhicule](https://trello.com/c/jf0R2Ttm/380-ajouter-un-nombre-de-place-pour-le-mod%C3%A8le-de-v%C3%A9hicule)

**Comment by Scttpr** [comment]
[Ajouter un nombre de place pour le modèle de véhicule](https://trello.com/c/jf0R2Ttm/380-ajouter-un-nombre-de-place-pour-le-mod%C3%A8le-de-v%C3%A9hicule)

File: models/car-model.mjs [hunk 1]
```diff
@@ -13,6 +13,11 @@ const CarModelSchema = new Schema({
   _id: String,
   label: {
     type: String,
+    unique: true,
```
Comment (Nainterceptor) [inline models/car-model.mjs]: It's not necessary to enforce label unicity.

File: models/car-model.mjs [hunk 2]
```diff
@@ -13,6 +13,11 @@ const CarModelSchema = new Schema({
...
+  },
+  capacity: {
+    type: Number,
+    required: true,
```
Comment (Nainterceptor) [inline models/car-model.mjs]: Mhhh maybe it's better to have a default value to prevent data migration ?

File: models/ride.mjs [hunk 3]
```diff
@@ -189,14 +197,37 @@ RideSchema.pre('validate', async function beforeSave() {
...
+      if (!carModel) {
+        const err = new Error();
+        err.status = 404;
+        err.message = 'Car model not found';
+        throw err;
+      }
+      if (!carModel.capacity) {
+        carModel.capacity = 3;
+      }
+      if (this.passengersCount > carModel.capacity || this.passengersCount > 3) {
```
Comment (Nainterceptor) [inline models/ride.mjs]: please copy capacity in nested document inside a ride

**Pull request merged**